src/Workspaces/Core/Portable/Shared/Extensions/SourceTextExtensions.cs

```diff
@@ -238,87 +236,6 @@ public static SourceText ReadFrom(ITextFactoryService textService, ObjectReader
         return textService.CreateText(textReader, encoding, checksumAlgorithm, cancellationToken);
     }
 
-    public static SourceText CreateSourceText(
```

Comment:
IDE impl that can be entirely removed now that the compiler side is just smarter.

Reply:
This also means we avoid the IDE making an entire copy of the node into our chunks, which then the compiler makes a copy of when making its LargeText instance itself.